.clj-kondo/config.edn

```diff
@@ -44,4 +44,9 @@
            :unresolved-var {:exclude [org.httpkit.client/get]}
            ;; for integration tests:
            :unused-namespace {:exclude [sample.unused.namespace
-                                        "more.unused.namespaces*"]}}}
+                                        "more.unused.namespaces*"]}}
+
+ :config-in-ns {;; for integration tests:
+                refactor-nrepl.ns.libspec-allowlist-test
+                {:linters {:unused-namespace {:exclude [really.important
```

Comment:
Let's rename `really.important` to `from-config-to-ns` so that we can see that the  `:config-in-ns` feature is getting code coverage